internal/controller/template_controller.go

```diff
@@ -49,7 +49,7 @@ import (
 type TemplateReconciler struct {
 	client.Client
 
-	downloadHelmChartFunc func(context.Context, *sourcev1.Artifact) (*chart.Chart, error)
+	DownloadHelmChartFunc func(context.Context, *sourcev1.Artifact) (*chart.Chart, error)
```

Comment:
I still see a lot of unnecessary changes to unrelated controllers, so I ask to get rid of them because there is no need for such changes at all. 

- `internal/controller/template_controller.go`
- `internal/controller/template_controller_test.go`
- `internal/controller/multiclusterservice_controller.go`
- `internal/controller/multiclusterservice_controller_test.go`
- `internal/controller/clusterdeployment_controller.go`
- `internal/controller/clusterdeployment_controller_test.go`